src/app/api/rooms/[roomId]/available/route.ts

```
@@ -13,6 +13,9 @@ export async function GET(
   try {
     const { searchParams } = new URL(request.url);
     const roomId = +params.roomId;
+    if (isNaN(roomId)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)